ceph-releases/luminous/ubuntu/16.04/daemon/docker_exec.sh

```diff
@@ -11,6 +11,7 @@ function set_trap_err {
 NOTRAP=
 function trap_error {
   set +x
+  declare -F err_cleanup && err_cleanup
```

Comment:
Please amend the commit message by explaining that people needed to exec a command _before_ the kill -TERM have to implement sigterm_clean_pre function and so fo after the kill and also for the ERR handling.

This have to be pretty well explained in the commit message

Reply:
done